### PR TITLE
Revamping Documentation

### PR DESCRIPTION
**Pull request by subhankarc**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -211,22 +279,7 @@ curl -sk -u broker:secret -H "X-Broker-Api-Version: 2.9" https://127.0.0.1:9293/
 cf service-access # should show all services as enabled, cf marketplace should show the same
 ```
 
-### Launch the Operators
-For running lifecycle operations, corresponding operator processes have to be started. Currently Service Fabrik Broker supports Bosh Director based and Docker Based services.
-
-Assuming that all required env variables (SETTINGS_PATH, NODE_ENV) are already set. Both bosh operator and docker operator can be launched.
-```shell
-node operators/StartBoshOperators.js #to start bosh operator
-node operators/StartDockerOperators.js #to start docker operator
```
Comment (jayeshjk) [inline README.md:220]: I think these also need to be adapted based on restructuring.

Comment (subhankarc) [reply README.md:220]: Yes, can you please suggest a change which I can commit ?

Comment (jayeshjk) [reply README.md:220]: For now,

```
node applications/operators/StartBoshOperators.js #to start bosh operator	
node application/operators/StartDockerOperators.js #to start docker operato
```
I think I will again have change due to https://github.com/cloudfoundry-incubator/service-fabrik-broker/pull/823

File: README.md [hunk 2]
```diff
@@ -170,36 +174,100 @@ If you need  to change the `settings.yml` configuration you should copy the file
...
 ```
+* Launch the Operators
+For running lifecycle operations, corresponding operator processes have to be started. Currently Service Fabrik Broker supports Bosh Director based and Docker Based services.
 
-### Launch the Broker
-[APIServer](https://kubernetes.io/docs/concepts/overview/kubernetes-api/) is a prerequisite for the Service Fabrik. To get more details about the APIs, [this](https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md) can be looked into.
+Assuming that all required env variables (SETTINGS_PATH, NODE_ENV) are already set. Both bosh operator and docker operator can be launched.
+```shell
+node operators/StartBoshOperators.js #to start bosh operator
+node operators/StartDockerOperators.js #to start docker operator
```
Comment (jayeshjk) [inline README.md:183]:
```suggestion
node applications/operators/StartBoshOperators.js #to start bosh operator
node applications/operators/StartDockerOperators.js #to start docker operator
```

File: README.md (hunk 1 above)
Comment (subhankarc) [reply README.md:220]: added your commit suggestions

File: README.md [hunk 3]
```diff
@@ -170,36 +174,100 @@ If you need  to change the `settings.yml` configuration you should copy the file
 # export SETTINGS_PATH=$(pwd)/deployment_hooks/config/my-settings.yml
 node $(pwd)/deployment_hooks/HookServer.js
```
Comment (jayeshjk) [inline README.md:175]:
```suggestion
node $(pwd)/applications/deployment_hooks/HookServer.js
```

File: README.md [hunk 4]
```diff
@@ -16,39 +19,74 @@ To facilitate plugging in external components, we intend to model service Fabrik
...
+helm install --set cluster.host=sf.ingress.< clusterdomain > --name interoperator --namespace interoperator helm-charts/interoperator
+```
 
-# Capabilities
+**NOTE:** `cluster.host` should be within the [63 character limit](http://man7.org/linux/man-pages/man7/hostname.7.html).
 
-1. Bringing in a new provisioner is easier.
 
-2. Bringing in new Backup and Restore approach and plugging in the existing framework is easier now.
+The default scheduler which gets deployed by the helm chart supports only one cluster with name "1". The scheduler type can be configured using `--set interoperator.config.schedulerType=<type>`  parameter to the helm install command. The current supported scheduler tyes are `default` and `round-robin`.
```
[the remark this answers is not in the document]
Comment (subhankarc) [reply README.md:60]: Good point......I remove the content here altogether

**Pull request merged**